build/lib/propertyInitOrderChecker.ts

```
@@ -0,0 +1,423 @@
+	| { readonly type: DefinitionKind.This; readonly node: ts.Node }
+	| { readonly type: DefinitionKind.String; readonly node: ts.StringLiteralLike }
+	| { readonly type: DefinitionKind.TripleSlashReference; readonly reference: ts.FileReference; readonly file: ts.SourceFile };
+
+/** @internal */
+export const enum EntryKind {
+	Span,
+	Node,
+	StringLiteral,
+	SearchedLocalFoundProperty,
+	SearchedPropertyFoundLocal,
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)